README.md

```diff
@@ -1,24 +1,65 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Blog App
+- This is a blog application that can have users write and post blogs, read blog posts, and edit or delete the posts.
 
-Things you may want to cover:
 
-* Ruby version
 
-* System dependencies
 
-* Configuration
+## Built With
 
-* Database creation
+  > Ruby/Rails
+  > PostgreSQL
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+## Getting Started
 
-* ...
+Follow the steps bellow to get a local copy up and running.
+
+- Run git clone https://github.com/gabrielcoder247/blog-app-rails.git from your terminal to clone the repo from GitHub.
+
+- Run cd blog-app to move to the project folder.
+
+-  Run yarn install from the command line to install dependencies.
+
+-  Run rails db:create, and rails db:migrate to create database.
+
+- Run rails db:seed to populate the database with the data in the seed file.
+
+-  Run rails s to run the local server
+
+- To get a local copy up and running follow these simple example steps.
+
+
+
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.